src/storage.js

```diff
@@ -74,6 +74,4 @@ class Storage {
   }
 }
 
-const instance = new Storage();
-
-export default instance;
+export default Storage;
```

Comment:
This change breaks compatibility. Is it safe to do it?

Reply:
This requires some changes in both wallets. Since right now we can't have a unique storage, I didn't find a way to keep compatibility, I had to return a class, instead of an instance.

In the wallets we call `hathorLib.storage.setStore(new LocalStorageStore());`, which used to make sense, because `hathorLib.storage` used to return a storage instance but right now it will return a class.

We also have direct storage calls, like `hathorLib.storage.setItem('wallet:version', VERSION);`, this would also require a refactor to support these changes.

Have you thought about a way I could keep compatibility?

Reply:
I think we can refactor our wallets as long as it is a simple task. If the refactor is not really simple, maybe we should keep exporting an instance instead of the class.

Reply:
Talking with @obiyankenobi about the `StorageProxy` I removed it and kept only our old storage and store classes. With this I don't need to refactor anything because it's still the same code for the storage.

Can you see if it's good?

Reply:
Seems good. Thanks!